requirements.txt

```diff
@@ -66,7 +66,6 @@ rfc2html>=2.0.1
 scout-apm>=2.23.0
 selenium>=3.141.0,<4.0
 six>=1.10.0
-svn>=1.0.1
 tblib>=1.3.0
 tqdm>=3.7.0
 #Trac>=1.0.10,<1.2
```

Comment:
Remove Trac entirely while we're at it?

Reply:
trac_links will go away when we refactor the /release view.
pulling the #Trac line out of requirements is reasonable.